raiden/network/proxies/service_registry.py

```diff
@@ -45,19 +53,36 @@ def __init__(
         self.client = jsonrpc_client
         self.node_address = self.client.address
 
-    def service_count(self, block_identifier: BlockSpecification) -> int:
-        """Get the number of registered services"""
-        result = self.proxy.contract.functions.serviceCount().call(
+    def ever_made_deposits(
+        self, block_identifier: BlockSpecification, index: int
+    ) -> Optional[Address]:
+        """Get one of the addresses that have ever made a deposit."""
+        try:
+            ret = Address(
+                to_canonical_address(
+                    self.proxy.contract.functions.ever_made_deposits(index).call(
+                        block_identifier=block_identifier
+                    )
+                )
+            )
+            assert isinstance(ret, T_Address)
```

Comment:
Why is the assertion here?

Reply:
Once I didn't see `mypy` catching this (I had an older version of `mypy` or something).  But now I've seen `mypy` working.  So I can remove this.